resources/scss/ceres/common/_common.scss

```diff
@@ -142,3 +148,7 @@ a:active {
 .border-radius-0 {
     border-radius: 0;
 }
+
+.border-top-0 {

```

Comment:
Ich würde diese Eigenschaft nicht in eine eigene Klasse auslagern.

Reply:
hatte es hier reingepackt, da border-radius-0 ebenfalls hier ist